core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitUtils.java

```diff
@@ -117,10 +117,10 @@ public static boolean isTransactionTableMetadata(TableMetadata tableMetadata) {
       //   - the column name without the "before_" prefix
       // if both columns don't exist, the table metadata is not transactional
       if (!tableMetadata.getColumnNames().contains(Attribute.BEFORE_PREFIX + nonPrimaryKeyColumn)
-          && !nonPrimaryKeyColumn.startsWith(Attribute.BEFORE_PREFIX)
-          && !tableMetadata
-              .getColumnNames()
-              .contains(nonPrimaryKeyColumn.substring(Attribute.BEFORE_PREFIX.length()))) {
+          && !(nonPrimaryKeyColumn.startsWith(Attribute.BEFORE_PREFIX)
+              && tableMetadata
+                  .getColumnNames()
+                  .contains(nonPrimaryKeyColumn.substring(Attribute.BEFORE_PREFIX.length())))) {
```

Comment:
```
nonPrimaryKeyColumn.startsWith(Attribute.BEFORE_PREFIX)
              && tableMetadata
                  .getColumnNames()
                  .contains(nonPrimaryKeyColumn.substring(Attribute.BEFORE_PREFIX.length())
 ```
                   
were meant to be written inside parentheses. 
This clause (line 120 to 123) verifies that if `nonPrimaryKeyColumn` is not an after image column (line 119), then it must be a before image column to be a transactional table metadata